zb-xiaomi.js

```diff
@@ -141,6 +141,79 @@ const MODEL_IDS = {
       },
     },
   },
+  'lumi.weather': {       // WSDCGQ11LM (square)
+    name: 'temperature',
+    // TODO: Change to TemperatureSensor once it display properly
+    '@type': ['MultiLevelSensor'],
```

Comment:
May I ask why we don't need `type: Constants.THING_TYPE_MULTI_LEVEL_SENSOR,` here?

Reply:
The use of 'type' is now obsolete, and has been replaced by '@type' which can have an array of types. When multiple types are present, then the user can choose which type is the "primary" type.

So for example, this should really look something like:
```
'@type': ['TemperatureSensor', 'HumiditySensor', 'PressureSensor']
```
then the user would select which of the 3 should be displayed on the "Things" page. You'll see all 3 when you look at the exploded view.

The reason I've chosen MultiLevelSensor for the time being is that we don't yet have support for the 3 different types right now in the UI. It will be coming soon.

Reply:
Thanks for the details :+1: